src/download.rs

```diff
@@ -172,17 +173,19 @@ fn write_image_and_sig(
 }
 
 /// Copy the image to disk and verify its signature.
+#[allow(clippy::too_many_arguments)]
 pub fn write_image<F>(
     source: &mut ImageSource,
     dest: &mut File,
     dest_path: &Path,
     image_copy: F,
     decompress: bool,
+    saved: Option<&SavedPartitions>,
```

Comment:
`saved_partitions` ?

Reply:
`saved` is the variable name everywhere else in the code.